packages/wdio-allure-reporter/src/index.js

```diff
@@ -193,7 +193,9 @@ class AllureReporter extends WDIOReporter {
             return
         }
 
-        if (this.options.disableWebdriverStepsReporting || this.options.useCucumberStepReporter || this.isMultiremote) {
```

Comment:
It seems that this patch disables whatever `useCucumberStepReporter` was suppose to be for, doesn't seem to me like a right solution.

Reply:
From my understanding, `useCucumberStepReporter` setting is used to change the report hierarchy when using cucumber. It goes this by using steps instead of tests (see line 103).

It appears it also disabled the command logging but as this is controlled using a  separate flag and I would like to use both features as per #6049 I think this is the correct approach.